tendrl-notifier.spec

```diff
@@ -66,6 +66,10 @@ py.test -v tendrl/notifier/tests || :
 %{_sysconfdir}/tendrl/notifier/email_auth.conf.yaml.sample
```

Comment:
entries for email_auth.conf.yaml.sample and snmp.conf.yaml files are repeated.
Hope we can remove ln 66 and 67.

Reply:
Will do that and push